Assets/Prefabs/Ped.prefab

```diff
@@ -551,3 +524,48 @@ Animation:
   m_PlayAutomatically: 1
   m_AnimatePhysics: 0
   m_CullingType: 0
+--- !u!1 &5725062646588365247
+GameObject:
```

Comment:
why do you need a separate CapsuleCollider ? Ped already has CharacterController which is actually CapsuleCollider with some new features.

Reply:
to prevent ped climb, slightly bigger collider prioritize to push object